utility_analysis_new/tests/dp_engine_test.py

```diff
@@ -167,6 +227,83 @@ def test_aggregate_error_metrics(self):
             for v in col
         ]
 
+    def test_multi_parameters(self):
+        # Arrange
+        aggregate_params = pipeline_dp.AggregateParams(
+            noise_kind=pipeline_dp.NoiseKind.GAUSSIAN,
+            metrics=[pipeline_dp.Metrics.COUNT],
+            max_partitions_contributed=1,
+            max_contributions_per_partition=1)
+
+        multi_param = utility_analysis_new.MultiParameterConfiguration(
+            max_partitions_contributed=[1, 2],
+            max_contributions_per_partition=[1, 2])
+
+        budget_accountant = pipeline_dp.NaiveBudgetAccountant(total_epsilon=1,
+                                                              total_delta=1e-10)
+
+        engine = dp_engine.UtilityAnalysisEngine(
+            budget_accountant=budget_accountant,
+            backend=pipeline_dp.LocalBackend())
+
+        # Input collection has 1 privacy id, which contributes to 2 partitions
+        # 1 and 2 times correspondingly.
+        input = [(0, 0), (0, 1), (0, 1)]
```

Comment:
very nit: why don't you use "pk0" and "pk1" as partition keys here but transform them with extractors? This seems to be a bit more straightforward.

Reply:
Good point! Done
I've updated in utility_analysis_test.py as well